programs/uxd/src/state/controller.rs

```diff
@@ -94,6 +101,13 @@ pub struct Controller {
     pub mercurial_vault_depository: Pubkey,
     pub credix_lp_depository: Pubkey,
 
+    // Flags needed for outflow limitation
+    pub outflow_limit_per_epoch_amount: u64,
+    pub outflow_limit_per_epoch_bps: u16,
+    pub seconds_per_epoch: u32,
```

Comment:
Not sure should it be seconds or slots, prob slots?

Reply:
Sounds good 👌  changed to slots_per_epoch and changed other places in the code to use the sysvar::Clock.slot instead